app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -87,10 +87,15 @@ def post_analytics(result)
     end
 
     def analytics_properties
+      parsed_phone = Phonelib.parse(phone)
+
       {
         context: context,
         multi_factor_auth_method: params[:otp_delivery_preference],
         confirmation_for_add_phone: confirmation_for_add_phone?,
+        area_code: parsed_phone.area_code,
+        country_code: parsed_phone.country,
+        phone_fingerprint: Pii::Fingerprinter.fingerprint(parsed_phone.e164),
```

Comment:
can we also document these keys? I think this is the `multi_factor_auth_enter_otp_visit` method

Reply:
I made a note to add them when I started, but it slipped my mind, thank you!